examples/internal/proto/examplepb/unannotated_echo_service.swagger.json

```diff
@@ -17,7 +17,12 @@
   },
   "tags": [
     {
-      "name": "UnannotatedEchoService"
+      "name": "UnannotatedEchoService",
+      "description": "UnannotatedEchoService description -- which should not be used in place of the documentation comment!",
+      "externalDocs": {
+        "description": "Find out more about UnannotatedEchoService",
+        "url": "https://github.com/grpc-ecosystem/grpc-gateway"
+      }
```

Comment:
Interesting. I guess this is correct? 😂.

Reply:
Changes to this file have been automatically generated by modifying the template.go. I think the tags defined in the following opanapi configuration file of `UnannotatedEchoService`, which were previously unsupported, now simply output.
https://github.com/grpc-ecosystem/grpc-gateway/blob/c12472a456fecdd504ccbb8eb645776c3c05f073/examples/internal/proto/examplepb/unannotated_echo_service.swagger.yaml#L70-L74

This might be the same as the output of the following tags defined in the a_bit_of_everything.proto example.
https://github.com/grpc-ecosystem/grpc-gateway/blob/c12472a456fecdd504ccbb8eb645776c3c05f073/examples/internal/proto/examplepb/a_bit_of_everything.proto#L508-L514

Do you have concerns about tags not being the expected output?

Reply:
No, I think it's fine :), I think you're right.